package.json

```diff
@@ -51,5 +54,8 @@
     "vue-router": "3.0.1",
     "vue-slick": "1.1.15",
     "vue-template-compiler": "2.6.10"
+  },
+  "lint-staged": {
+    "*.{js,vue,ts}": "vue-cli-service lint"
```

Comment:
Only thing I noticed is that we ignore the node_modules folder in freshebt-vue. Not sure if that is necessary here since it's a much newer version.

Reply:
Interesting... `lint-staged` only tries to run the command on files that are staged in git (i guess that's kinda obvious from the name). I think the freshebt one is a little off, since anything in the .gitignore including node_modules can't be staged.